cmd/siklu/bank_management.c

```diff
@@ -54,6 +56,27 @@ struct software_bank_t* bank_management_get_current_bank(void) {
 		goto fail_free;
 	}
 	
+	config_boot_tries_left = siklu_fdt_getprop_string(fdt, "/", PROP_BOOT_TRIES_LEFT, NULL);
+	if (IS_ERR(config_boot_tries_left)) {
+		printf("Could not read boot_tries_left\n");
```

Comment:
missing goto?

Reply:
Default counter value is negative, so whole feature is ignored then

Reply:
now it returns here